VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1352,61 +1354,34 @@ private AccessibilityManager getAccessibilityManager() {
      */
     private boolean getTouchExploreEnabled() {
         boolean ret = false;
-        AccessibilityManager am = getAccessibilityManager();
-        if (am != null && am.isTouchExplorationEnabled()) {
-            ret = true;
-        }
-        return ret;
-    }
 
-    /**
-     * @param am The accessibility manager to unregister from
-     * @return True if successful
-     */
-    private boolean unregisterTouchExploreChangeListener(@NonNull AccessibilityManager am) {
-        boolean ret = false;
-        if (touchExplorationStateChangeListener == null) {
-            ret = true;
-        }
-        else if (am.removeTouchExplorationStateChangeListener(touchExplorationStateChangeListener)) {
-            touchExplorationStateChangeListener = null;
-            ret = true;
+        if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+            AccessibilityManager am = getAccessibilityManager();
```

Comment:
Why can't we use a global variable for the accessibilityManager? Seems like we explicitly call for this in many places.